resource_customizations/argoproj.io/Application/actions/toggle-auto-sync/action.lua

```diff
@@ -0,0 +1,18 @@
+function toggleAutoSync(obj)
+    if obj.spec.syncPolicy and obj.spec.syncPolicy.automated then
+        obj.spec.syncPolicy.automated = nil
```

Comment:
Missing a check to set `obj.spec.syncPolicy.automated.enabled = false` if it is currently true